src/actions.js

```diff
@@ -1055,3 +1061,37 @@ export const nanoContractHistoryFailure = (payload) => ({
   type: types.NANOCONTRACT_HISTORY_FAILURE,
   payload,
 });
+
+/**
+ * Request unregistration of a Nano Contract by its key.
+ * @param {{
+ *   ncId: string,
+ * }} Nano Contract ID to unregister.
+ */
+export const nanoContractUnregisterRequest = (unregisterRequest) => ({
+  type: types.NANOCONTRACT_UNREGISTER_REQUEST,
+  payload: unregisterRequest,
+});
+
+/**
+ * Unregistration of a Nano Contract has finished with success.
+ * @param {{
+ *   ncId: string,
+ * }} Nano Contract ID unregistered.
+ */
+export const nanoContractUnregisterSuccess = (unregistered) => ({
+  type: types.NANOCONTRACT_UNREGISTER_SUCCESS,
+  payload: unregistered,
+});
+
+/**
+ * Request a change on the Nano Contract registered.
+ * @param {{
+ *   newAddress: string;
+ *   oldNc: Object;
```

Comment:
Why do you need the full object, not only the `ncId`?

Reply:
Changed to ncId.